### PR TITLE
Email Communications Front-end

### PR DESCRIPTION
**Pull request by sam-warren**
## Status
**Ready**

## Fixes or Implements [AB#8195](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8195)
- [X] Enhancement
- [ ] Bug
- [ ] Documentation

## Description:
- Added email creation fields including a WYSIWYG editor to the communications page so an admin can send out a mass email to all HG users.
- Extracted comms table into its own separate component

### UI Changes
YES

### Browsers Tested
- [X] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

Sample Email:
Screen Shot 2020-07-09 at 5 20 58 PM

Confirmation Feedback:
Screen Shot 2020-07-09 at 5 21 07 PM

### CODE REVIEW

File: Apps/AdminWebClient/src/ClientApp/src/App.vue [hunk 1]
```diff
@@ -2,9 +2,9 @@
     <v-app id="healthGatewayAdmin">
         <ToolbarComponent />
         <MainNavbar title="HealthGateway Admin" />
-        <v-main>
+        <v-content>
```
Comment (tiagograf-AOT) [inline Apps/AdminWebClient/src/ClientApp/src/App.vue:5]: can you explain why this?

Comment (FuriousLlama) [reply Apps/AdminWebClient/src/ClientApp/src/App.vue:5]: Those tags where deprecated and where showing warnings. v-content is the new tag

File: Apps/AdminWebClient/src/ClientApp/src/components/core/EmailCommunication.vue [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            this.showFeedback = true;
+            this.bannerFeedback = {
+                type: ResultType.Success,
+                title: "Success",
+                message: "Email sent."
+            };
+            this.subject = "";
+            this.priority = "";
+            this.content = "<p></p>";
+            (this.$refs.form as Vue & {
```
Comment (FuriousLlama) [inline Apps/AdminWebClient/src/ClientApp/src/components/core/EmailCommunication.vue:147]: Is there a type that has said method? if so lets cast it to that instead

Comment (sam-warren) [reply Apps/AdminWebClient/src/ClientApp/src/components/core/EmailCommunication.vue:147]: Not from my research

**Pull request merged**